### PR TITLE
add 「Test the installation」 for go-get function

### PR DESCRIPTION
**Pull request by AzenXu**
Please read https://github.com/etcd-io/etcd/blob/master/CONTRIBUTING.md#contribution-flow.

### CODE REVIEW

File: Documentation/dl_build.md [hunk 1]
```diff
@@ -48,14 +48,18 @@ $ $GOPATH/bin/etcd
...
 $ ETCDCTL_API=3 ./bin/etcdctl put foo bar
 OK
 ```
 
-If OK is printed, then etcd is working!
+If etcd is built using GOPATH, run the following:
+```
+$ cd $GOPATH/src/go.etcd.io/etcd/etcdctl && go build . && mv etcdctl $GOPATH/bin && ETCDCTL_API=3 $GOPATH/bin/etcdctl put foo bar
+OK
+```
```
Comment (hexfusion) [inline Documentation/dl_build.md]: Can we simplify this with `go install`? also If $PATH contains $GOPATH/bin `etcdctl` will work without the prefix.

**Comment by hexfusion** [comment]
@AzenXu thanks for taking the time to do this PR I added a small note, also can you also reformat your git commit to fix the travis error? Maybe something like `Documentation: add go install example for dl_build` ?

**Comment by wenjiaswe** [comment]
@jingyih

File: Documentation/dl_build.md [hunk 2]
```diff
@@ -48,14 +48,18 @@ $ $GOPATH/bin/etcd
 
 ### Setting a key
 
-Run the following:
+If etcd is built without using GOPATH, run the following:
```
Comment (xiang90) [inline Documentation/dl_build.md]: i do not really understand this. how to run/build etcd has nothing to do with GOPATH.

Comment (jingyih) [reply Documentation/dl_build.md]: I think this is referring to the previous section, where two methods were provided to build the latest version of etcd:
1. To build etcd from the master branch **without a GOPATH** using the official build script
2. To build a vendored etcd from the master branch via go get, where it says **GOPATH should be set**

**Pull request closed without merging**